CMakeLists.txt

```diff
@@ -247,6 +249,48 @@ if(SERIALBOX_TESTING)
   endif(SERIALBOX_ENABLE_C)
 endif(SERIALBOX_TESTING)
 
+#---------------------------------------- pFUnit ---------------------------------------------------
+if(${SERIALBOX_TESTING_FORTRAN})

```

Comment:
Is there a reason to have the `if(${SERIALBOX_TESTING_FORTRAN})` instead of `if(SERIALBOX_TESTING_FORTRAN)` as you have later?